validator/sawtooth_validator/journal/chain_commit_state.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2016 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+
+class _CommitCache(object):
+    """Tracks the committment status of a set of identifiers in these
+    identifiers are either explicitly committed, explicity uncommitted, if
+    they fall in to neither of these cases then the fallback is to look in
+    the BlockStore to see if they are there. Explicit committed ids
```

Comment:
Suggest changing first sentence to the following:

Tracks the commit status of a set of identifiers and these
identifiers are either explicitly committed, or explicitly uncommitted. If
they fall in to neither of these cases then the fallback is to look in
the BlockStore to see if they are there.